JuiceMaker/JuiceMaker/Controller/JuiceOrderViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  JuiceMaker - ViewController.swift
+//  Created by yagom. 
+//  Copyright © yagom academy. All rights reserved.
+// 
+
+import UIKit
+
+final class JuiceOrderViewController: UIViewController {
+    @IBOutlet var fruitLabels: [UILabel]!
+    
+    private let recipe: [JuiceMaker.Menu: [(Fruit, Int)]] = ([.strawberryJuice         : [(.strawberry, 16)],
```

Comment:
```suggestion
    private let recipe: [JuiceMaker.Menu: JuiceMaker.Recipe] = ([.strawberryJuice         : [(.strawberry, 16)],
```

이렇게 타입 명시를 할 수도 있을 것 같습니다.